### PR TITLE
Recaptcha

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "depd": "1.1.2",
     "setprototypeof": "1.1.0",
     "statuses": "1.5.0",
-    "http-errors": "1.6.3"
+    "http-errors": "1.6.3",
+    "ms": "2.0.0"
```
Comment (randyhoulahan) [inline package.json:39]: was this used?

**Review by randyhoulahan — APPROVED** [review]
MS lib added.  Was it used?

File: app/views/password-reset.js [hunk 2]
```diff
@@ -1,20 +1,21 @@
-define(['app', 'directives/input-email'], function(){
+define(['app', 'directives/input-email', './recaptcha.directive.js'], function(grecaptcha){
```
Comment (sbilodeau) [inline app/views/password-reset.js:1]: is .js required?
grecaptcha will contains `app`

File: app/views/password-reset.js [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 
-return ['$scope', '$http', '$location', function ($scope, $http, $location) {
+return ['$scope', '$http', '$location', '$q', function ($scope, $http, $location, $q) {
```
Comment (sbilodeau) [inline app/views/password-reset.js:3]: no use of `$q`

File: app/views/signup.js [hunk 4]
```diff
@@ -1,19 +1,38 @@
-define(['app', 'directives/input-email', 'directives/security/password-rules'], function(){
+define(['app', 'directives/input-email', 'directives/security/password-rules', './recaptcha.directive.js'], function(){
```
Comment (sbilodeau) [inline app/views/signup.js:1]: should `.js` be removed?

File: app/views/signup.js [hunk 5]
```diff
@@ -1,19 +1,38 @@
...
 
-return ['$scope', '$http', 'authentication', 'apiToken', 'user', function ($scope, $http, authentication, apiToken, user) {
+return ['$scope', '$http', 'authentication', 'apiToken', 'user', '$q', function ($scope, $http, authentication, apiToken, user, $q) {
```
Comment (sbilodeau) [inline app/views/signup.js:3]: no use of `$q`

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**